src/org/safs/rest/service/models/consumers/SafsrestAdapter.groovy

```diff
@@ -276,17 +264,6 @@ class SafsrestAdapter extends ClientPOJOAdapter {
         "SAFSREST${useScript ? '_using_OS_shell' : ''}${execCurlFromJVM ? '_execCurlFromJVM' : ''}"
     }
 
-    public setAuth(auth) {
```

Comment:
This will make SafsrestAdapter.groovy independent from Auth, it is the good side.
But the class Service will become heavier, and if we want to support the OAuth2 for other Adapters, we have to modify the Service class again and again.